coordinator/tests/debug/conversions.spec.ts

```diff
@@ -11,7 +11,7 @@ afterAll(() => {
 })
 
 const allExpectedPassingConvertExamples = fetcher.convertExamples.filter(e => e.isSuccess)
-test.skip.each(allExpectedPassingConvertExamples)(
+test.each(allExpectedPassingConvertExamples)(
```

Comment:
undo!!!!!!!!!!!!!!!!!!!!!

Reply:
!!!!!!! Noooooooooo! I can't believe I've done this